src/transformState.js

```diff
@@ -4,8 +4,29 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  const trans = state;
```

Comment:
Do you really need it? In this task you should change the original state.
![image](https://user-images.githubusercontent.com/96409563/206140261-da3b8ffd-15cf-486b-8aed-41085d188690.png)
Btw, if you wanted to create a copy of object, it is not working like that 
https://www.javascripttutorial.net/object/3-ways-to-copy-objects-in-javascript/